Musakai.Blazor/Pages/Home.razor

```diff
@@ -32,6 +33,39 @@
     <PortfolioCard Src="" Title="Street Fotografie" />
     <PortfolioCard Src="" Title="Svatby" />
 </section>
+<section id="contact">
+    <div class="contact-info">
+        <h2>Contact Information</h2>
+        <p><strong>Email:</strong> example@example.com</p>
```

Comment:
Not sure if the email should be computer-readable as it can be easily scraped.

Reply:
You're right but our goal is to prepare HTML structure here.

Reply:
Maybe add comment there.

Reply:
Wrote to Lipsum